src/policy/policy.cpp

```diff
@@ -142,8 +142,12 @@ bool IsStandardTx(const CTransaction& tx, const std::optional<unsigned>& max_dat
             reason = "bare-multisig";
             return false;
         } else if (IsDust(txout, dust_relay_fee)) {
-            reason = "dust";
-            return false;
+            // TODO: Ensure ephemeral output is spent in the same mempool package
+            // to avoid bloating utxo set
+            if (whichType != TxoutType::ANCHOR) {
```

Comment:
Do you want to allow *non-dust* outputs to be anchor?

Reply:
Yes that's the idea. This allows protocols to create outputs to dump excess value, instead of dumping it to fees(which complicates future V3/eviction policies).

Reply:
Can you elaborate, or send me a link if it's already summarized somewhere online?

Reply:
Hmm, looks like I don't discuss it in the bip draft: https://github.com/instagibbs/bips/blob/ephemeral_anchor/bip-ephemeralanchors.mediawiki

Basically, the design choices should be either:
1) it must be 0-value
2) it can be any value

Choosing (2) allows any value that would have gone to "trimmed outputs" in things like LN to go to those outputs, instead of to the commitment transaction fee, since the EA requirement of 0-fee EA transaction exists to make eviction reasoning easier.

See https://github.com/instagibbs/bolts/blob/eltoo_draft/XX-eltoo-transactions.md#ephemeral-anchor-output for an example usage.

Reply:
switch the ordering? `else if (whichType != ANCHOR && IsDust())`

Reply:
reworked @ajtowns